lib/rules/render-result-naming-convention.ts

```diff
@@ -51,9 +52,12 @@ export default ESLintUtils.RuleCreator(getDocsUrl)({
     let renderAlias: string | undefined;
     let wildcardImportName: string | undefined;
 
+    const testingLibraryRenderName = getTestingLibraryRenderImport(context);
```

Comment:
This is how I'm planning to use the new methods, so you can just do this:

```javascript
create(context) {
  const testingLibraryRenderName = getTestingLibraryRenderImport(context);
  // now you have 'render' within this var if render found and imported from related Testing Library module

  return {
    [`VariableDeclarator CallExpression[callee.name=${testingLibraryRenderName}]`]() {
      // here you don't need to check anything manually about render alias or imported properly anymore!
      // just implement the actual checks for the rule
    }
  }
}
```

Reply:
I really like this

Reply:
Right? I hope I can implement this version, so it would be really easy to focus on specific rules implementations. This is why is taking me so long basically, I'm trying to 1) find proper approach and 2) figure how to implement it. There are couple of things I need to try with `context.getSourceCode`, I think I might have something.